src/Provider.tsx

```diff
@@ -105,7 +105,7 @@ export function configureStore<ClientState, ClientActions extends Action<any>>(
     // TODO: Extract this to an utility
     const reducers = { ...coreReducers } as CombinedReducersMapObject<CoreStore & ClientState, CombinedActions>
     Object.keys(customReducers).forEach((reducerName: Extract<keyof ClientState, string>) => {
-        const coreInitialState = coreReducers[reducerName]
+        const coreInitialState = coreReducers[reducerName](undefined, {type: ' UNKNOWN ACTION '})
```

Comment:
Null coalescing should be applied or we'll crash the app when we process a reducer slice which is not part of our core (store.getState().customSlice). 

Good catch otherwise. I'm unsure how it should've been written to avoid this kind of bug:  functions are object type hence allowed to be spread, and it seems that customReducers' initial state pass typings due to extending nature of the client slice over the core slice. I'd say overall typings for the provider should be improved.

Reply:
Changed to
```const coreInitialState = coreReducers[reducerName]?.(undefined, {type: ' UNKNOWN ACTION '})```